dotnet/metadata.yaml

```diff
@@ -0,0 +1,19 @@
+# High-value example
```

Comment:
You need section separators around each files block: --- (3 dashes) starts a section and ... (3 dots) ends a section.

```
---
files: <>
...
---
files: <>
...
```

Reply:
Also, be sure to list any .cs supporting files that don't contain service calls (like test files) with just the path and no services block. Otherwise, they'll be picked up as not yet cleaned.

Reply:
Just updated and pushed the changes (with more tweaks).  Please review again.  :)

Reply:
Looks good!